chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -61,6 +64,20 @@
   include_recipe "ceph::radosgw_keystone"
 end
 
+admin_tenant_id = keystone_settings["admin_tenant_id"]
+
+# add caps to user admin to access the bucketst 
+if node.roles.include?("ceph-radosgw")
```

Comment:
I think this is reduntant. As far as I can see the radosgw recipe is only ever run on nodes that actually have the radosgw-role.